nbdime/prettyprint.py

```diff
@@ -6,19 +6,137 @@
 from __future__ import unicode_literals
 from __future__ import print_function
 
-from six import string_types
+from itertools import chain
+import os
 import pprint
+import re
+from subprocess import Popen, PIPE
+import tempfile
+from textwrap import indent
```

Comment:
Python 2.7 does not support indent. Introduced in 3.3. There is an initial_indent and subsequent_indent in 2.7.